positron/modules/atom_browser_web_contents.js

```diff
@@ -29,6 +32,57 @@ let WebContents_prototype = {
   loadURL: function(url) {
     this._browserWindow._domWindow.location = url;
   },
+
+  openDevTools: function() {
```

Comment:
Over in #11, @marcoscaceres reminded me about method definitions, which would make this definition simpler.